src/main/java/com/google/jenkins/plugins/computeengine/InstanceConfiguration.java

```diff
@@ -783,6 +826,11 @@ public FormValidation doCheckMachineType(@QueryParameter String value) {
             return FormValidation.ok();
         }
 
+        @SuppressWarnings("unused")
+        public ProvisioningType defaultProvisioningType() {
+            return new Standard();
```

Comment:
there is a bug in here,

it should return `new Preemptible()` if the `preemptible` field was already set when using the previous version of the plugin..

(I am fixing it already)

Reply:
fixed in https://github.com/jenkinsci/google-compute-engine-plugin/pull/492/commits/e806d2550e178cc3b4525e544957d93fb5dbaf9a